### PR TITLE
Initial product elastic migration

### PR DESCRIPTION
**Pull request by alvian-machtwatch**
## Description & Technical Solution
For migrating from postgre to elastic for product schema

## How to Try Locally
Read the readme

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+.PHONY: setup
+setup: # set up the necessary dependencies and configurations required for development
+	@echo "\n🔎 Setting up repo requirements...\n"
+	@.dev/setup.sh
+
+.PHONY: env
+env: # generate development .env files from vault 
+	@echo "Generating .env file.."
+	@.dev/generate-env.sh $(GITHUB_TOKEN) $(service_name) development
```
Comment (kumirz) [inline Makefile:21]:
```suggestion
.PHONY: env-local
env: # generate local .env files from vault 
	@echo "Generating local .env file.."
	@.dev/generate-env.sh $(GITHUB_TOKEN) $(service_name) local
	
.PHONY: env-dev
env: # generate development .env files from vault 
        @echo "Generating development .env file.."
        @.dev/generate-env.sh $(GITHUB_TOKEN) $(service_name) development

.PHONY: env-staging
env: # generate staging .env files from vault 
        @echo "Generating staging .env file.."
        @.dev/generate-env.sh $(GITHUB_TOKEN) $(service_name) staging
```

File: config/config.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+	return false
+}
+
+// SetConfig will set config from .env file if it's exist
+//
+// Otherwise it will set from system's ENV variables.
+// Filename should be and env file: .env or .env.* file.
+// Dirpath should be in this format: /some/dirpath.
+func SelectConfig(environment string) error {
```
Comment (kumirz) [inline config/config.go:40]: ini dipakek ga mas?

**Review by kumirz — APPROVED** [review]
aman, nitip komen dikit

File: config/config.go (hunk 2 above)
Comment (alvian-machtwatch) [reply config/config.go]: tidak mas, dah dibuang yak

**Pull request merged**